### ISSUE TITLE
Does Horovod means the larger batchsize like the ordinary distributed Tensorflow?

### ISSUE THREAD

**Issue by darchonyzx** [horovod/horovod#773]

**Comment by mr-ubik** [comment]
Since the documentation says to change the learning rate to compensate for the batch size increase, I would hazard a yes.

**Comment by tgaddair** [comment]
@darchonyzx, I'm not sure which distributed TensorFlow policy you're referring to.  In your script, the batch size is going to be given per-device.  So in your case: 8.  However, the _effective_ batch size will be 32 because you're running 4 processes and aggregating the results.
